README.md

```diff
@@ -65,9 +65,11 @@
 This is a fork of Prettier that adds a new command line option `--paren-spacing` which inserts many extra spaces inside parentheses, the way how projects in the WordPress ecosystem (Calypso, Gutenberg, etc.) like to format their code.
 
 In order to install a version based on a particular upstream version (like 1.x.x), run
+
 ```sh
 npm i --save-dev "git+https://github.com/Automattic/wp-prettier.git#wp-prettier-1.x.x"
```

Comment:
I know it's not new, but this is stale. It should now be something like `"prettier": "npm:wp-prettier@latest"`

Reply:
Updated